integration-tests/src/test/resources/negative-cases/empty_expression_value.proto

```diff
@@ -26,26 +26,20 @@
 
 syntax = "proto3";
 
-package spine.protodata.hello;
+package spine.examples.protodata.hello;
 
 import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
-option java_package = "io.spine.protodata.hello";
-option java_outer_classname = "EchoProto";
+option java_package = "io.spine.examples.protodata.hello";
+option java_outer_classname = "SizeOptionTestProto";
 option java_multiple_files = true;
 
-import "google/protobuf/timestamp.proto";
-import "google/protobuf/wrappers.proto";
-import "google/protobuf/any.proto";
+import "spine/examples/protodata/hello/options.proto";
 
-message Echo {
+message SizeOptionTest {
```

Comment:
To avoid confusion with a test class (i.e. the one with test methods), let's name it `SizeOptionTestMsg`.